### PR TITLE
NAS-102297 / 11.3 / Bug fix for multiple ip's in vnet

### PR DESCRIPTION
**Pull request by sonicaj**
This commit fixes a bug where multiple ip's donot work with vnet on.

### CODE REVIEW

File: iocage_lib/ioc_start.py [hunk 1]
```diff
@@ -1181,7 +1181,7 @@ def start_network_vnet_addr(self, iface, ip, defaultgw, ipv6=False):
         if ipv6:
             ifconfig = [iface, 'inet6', ip, 'up']
         else:
-            ifconfig = [iface, ip, 'up']
```
Comment (skarekrow) [inline iocage_lib/ioc_start.py:1184]: We need to only alias on VNET when the initial IP has been set. We don't want them all to be aliases

**Review by skarekrow — CHANGES_REQUESTED** [review]

File: iocage_lib/ioc_start.py (hunk 1 above)
Comment (william-gr) [reply iocage_lib/ioc_start.py:1184]: What do you mean, everything is an alias. There is no such thing as primary IP

File: iocage_lib/ioc_start.py [hunk 2]
```diff
@@ -1181,7 +1181,7 @@ def start_network_vnet_addr(self, iface, ip, defaultgw, ipv6=False):
...
+            ifconfig = [iface, ip, 'alias']
```
Comment (william-gr) [inline iocage_lib/ioc_start.py:1184]: You still need the up and the end

Comment (sonicaj) [reply iocage_lib/ioc_start.py:1184]: Nope. Docs say that when ever an ip is added, the interface is automatically changed to `up` state

**Review by william-gr — APPROVED** [review]

File: iocage_lib/ioc_start.py (hunk 1 above)
Comment (skarekrow) [reply iocage_lib/ioc_start.py:1184]: As far as I know there is a difference. Aliases have a broadcast on just their IP instead of the whole subnet.

Comment (sonicaj) [reply iocage_lib/ioc_start.py:1184]: @skarekrow i haven't found anything which supports that ?
```
inet 192.168.0.26 netmask 0xffffff00 broadcast 192.168.0.255
net 192.168.0.27 netmask 0xffffff00 broadcast 192.168.0.255
```
Please advise. Also i am not sure what benefit we would have of having broadcast at all in that case ?

**Review by skarekrow — APPROVED** [review]

**Pull request merged**